src/components/implementation/no_interface/cbboot/quarantine.c

```diff
@@ -401,14 +445,16 @@ fault_quarantine_handler(spdid_t spdid, long cspd_dspd, int cap_ccnt_dcnt, void
 
 		/* FIXME: which fork should be used here? This pulls the most
 		 * recent fork. */
-		f_spd = LAST_LIST(d, n, p)->fork_spd;
+		f_spd = 14; // LAST_LIST(d, n, p)->fork_spd;
```

Comment:
this is supposed to be a generic server.  We don't want hardcoded spdids.

Please removed commented code that should NOT be in a PR.